prime-router/src/test/kotlin/azure/MessagesFunctionsTests.kt

```diff
@@ -284,7 +290,7 @@ class MessagesFunctionsTests {
         )
 
         val unAuthRes = messagesFunctions.messageSearch(unAuthReq)
-        assert(unAuthRes.status.equals(HttpStatus.UNAUTHORIZED))
+        assertThat(unAuthRes.status).isEqualTo(HttpStatus.UNAUTHORIZED)
```

Comment:
Made this change to help debug this test and align with the rest of the code base